main/guides/js-programming/hardened-js.md

```diff
@@ -79,19 +79,19 @@ e.g., only giving the `entryGuard` the ability to increment the counter.
 
 This limits the damage that can happen if there is an exploitable bug.
 
-### Widely Shared vs. Closely Held
+### [Widely Shared vs. Closely Held](/guides/zoe/contract-access-control.html#access-control-with-objects)
```

Comment:
sorry; I meant the other way around: there are uses of these terms under "access control with objects". Please link those uses to the definitions here.

Also: Headings should not be links.

Reply:
I made the update. Lmk what you think